src/handlers/ssh_authorized_keys.rs

```diff
@@ -283,14 +287,26 @@ pub async fn rename_authentication_key(
         .ok_or(WebError::DbError("Returned user had no ID".into()))?;
     if let Some(mut key) = AuthenticationKey::find_by_id(&appstate.pool, key_id).await? {
         if key.yubikey_id.is_some() {
+            error!(
```

Comment:
```suggestion
            warn!(
```

Reply:
Error logs should be thrown when encountering invalid application state, execution errors, misconfiguration etc. This is regular application operation.